docker/0.23-1/base/Dockerfile.cpu

```diff
@@ -1,9 +1,11 @@
 ARG UBUNTU_VERSION=18.04
+ARG UBUNTU_IMAGE_DIGEST=646942475da61b4ce9cc5b3fadb42642ea90e5d0de46111458e100ff2c7031e6
 
-FROM ubuntu:${UBUNTU_VERSION}
+FROM ubuntu:${UBUNTU_VERSION}@sha256:${UBUNTU_IMAGE_DIGEST}
```

Comment:
We're locking the ubuntu image? This might lead to not picking up security patches if a vulnerability is found in an older image.

Reply:
Yes, I argue that if a vulnerability is found, we have to find the updated sha that has the patches and have an engineer review the update. I believe the solution for such a situation is to have a process for scanning the images, not to have the linux dependencies unpinned. Too many unknowns with even the linux dependencies being updated unknowingly as I experienced with the CMAKE update breaking MLIO.

Reply:
Gotcha, makes sense.